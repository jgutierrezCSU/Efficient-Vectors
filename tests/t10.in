mergeKqueues
5
20	1	6	6	10	12	16	17	18	23	31	32	33	33	35	40	41	41	47	49	50
10	3	8	11	17	19	23	25	44	45	46
15	2	5	7	11	17	19	19	19	20	24	35	36	41	42	46
25	2	2	5	8	13	15	20	22	24	25	26	28	28	29	36	40	40	41	42	43	44	44	47	48	49
13	3	5	9	15	15	17	23	29	35	36	40	47	49

