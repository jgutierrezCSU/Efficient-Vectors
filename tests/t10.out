After merging of 5 queues, the resulting vector is:
1 2 2 2 3 3 5 5 5 6 6 7 8 8 9 10 11 11 12 13 15 15 15 16 17 17 17 17 18 19 19 19 19 20 20 22 23 23 23 24 24 25 25 26 28 28 29 29 31 32 33 33 35 35 35 36 36 36 40 40 40 40 41 41 41 41 42 42 43 44 44 44 45 46 46 47 47 47 48 49 49 49 50 
